Cargo.toml

```diff
@@ -688,35 +688,35 @@ visibility = "0.1.1"
-# alloy-rpc-types-engine = { git = "https://github.com/alloy-rs/alloy", rev = "cfb13aa" }
-# alloy-rpc-types-eth = { git = "https://github.com/alloy-rs/alloy", rev = "cfb13aa" }
-# alloy-rpc-types-mev = { git = "https://github.com/alloy-rs/alloy", rev = "cfb13aa" }
-# alloy-rpc-types-trace = { git = "https://github.com/alloy-rs/alloy", rev = "cfb13aa" }
-# alloy-rpc-types-txpool = { git = "https://github.com/alloy-rs/alloy", rev = "cfb13aa" }
-# alloy-serde = { git = "https://github.com/alloy-rs/alloy", rev = "cfb13aa" }
-# alloy-signer = { git = "https://github.com/alloy-rs/alloy", rev = "cfb13aa" }
-# alloy-signer-local = { git = "https://github.com/alloy-rs/alloy", rev = "cfb13aa" }
-# alloy-transport = { git = "https://github.com/alloy-rs/alloy", rev = "cfb13aa" }
-# alloy-transport-http = { git = "https://github.com/alloy-rs/alloy", rev = "cfb13aa" }
-# alloy-transport-ipc = { git = "https://github.com/alloy-rs/alloy", rev = "cfb13aa" }
-# alloy-transport-ws = { git = "https://github.com/alloy-rs/alloy", rev = "cfb13aa" }
+[patch.crates-io]
+alloy-consensus = { git = "https://github.com/alloy-rs/alloy", rev = "997e211" }
+alloy-contract = { git = "https://github.com/alloy-rs/alloy", rev = "997e211" }
+alloy-eips = { git = "https://github.com/alloy-rs/alloy", rev = "997e211" }
+alloy-genesis = { git = "https://github.com/alloy-rs/alloy", rev = "997e211" }
+alloy-json-rpc = { git = "https://github.com/alloy-rs/alloy", rev = "997e211" }
+alloy-network = { git = "https://github.com/alloy-rs/alloy", rev = "997e211" }
+alloy-network-primitives = { git = "https://github.com/alloy-rs/alloy", rev = "997e211" }
+alloy-node-bindings = { git = "https://github.com/alloy-rs/alloy", rev = "997e211" }
+alloy-provider = { git = "https://github.com/alloy-rs/alloy", rev = "997e211" }
+alloy-pubsub = { git = "https://github.com/alloy-rs/alloy", rev = "997e211" }
+alloy-rpc-client = { git = "https://github.com/alloy-rs/alloy", rev = "997e211" }
+alloy-rpc-types = { git = "https://github.com/alloy-rs/alloy", rev = "997e211" }
+alloy-rpc-types-admin = { git = "https://github.com/alloy-rs/alloy", rev = "997e211" }
+alloy-rpc-types-anvil = { git = "https://github.com/alloy-rs/alloy", rev = "997e211" }
+alloy-rpc-types-beacon = { git = "https://github.com/alloy-rs/alloy", rev = "997e211" }
+alloy-rpc-types-debug = { git = "https://github.com/alloy-rs/alloy", rev = "997e211" }
+alloy-rpc-types-engine = { git = "https://github.com/alloy-rs/alloy", rev = "997e211" }
+alloy-rpc-types-eth = { git = "https://github.com/alloy-rs/alloy", rev = "997e211" }
+alloy-rpc-types-mev = { git = "https://github.com/alloy-rs/alloy", rev = "997e211" }
+alloy-rpc-types-trace = { git = "https://github.com/alloy-rs/alloy", rev = "997e211" }
+alloy-rpc-types-txpool = { git = "https://github.com/alloy-rs/alloy", rev = "997e211" }
+alloy-serde = { git = "https://github.com/alloy-rs/alloy", rev = "997e211" }
```

Comment:
Using a patch of alloy: commit comes from the `main` branch [here](https://github.com/alloy-rs/alloy/tree/997e211457c4499e52f34ad262f98a9774ce0840)

Reply:
reverted this patch, now waiting for https://github.com/paradigmxyz/reth/pull/15635 to merge